docker/servicesmw/Jenkinsfile

```diff
@@ -0,0 +1,195 @@
+        }
+      }
+    )
+
+    imageTag = "$appHash.$configHash"
+
+    println("Image tag: $imageTag")
+  }
+
+  stage("Bump cachebuster and version") {
+    parallel (
+      "app": {
+        dir("app") {
+          def cacheBuster = new Date().getTime()
+
+          sh('echo "<?php\n\\\$wgStyleVersion = ' + cacheBuster + ';" > wgStyleVersion.php')
+          sh('echo "dev@' + appHash + '" > wikia.version.txt')
+        }
+      },
+
+      "config": {
+        dir("config") {
+          sh('echo "dev@' + configHash + '" > wikia.version.txt')
+        }
+      }
+    )
+  }
+
+  parallel (
+    "php": {
+      stage("Build and push php image") {
+        dir("app") {
+          def imageExists = false
+          def status = sh(script: """
+            curl -u ${env.JENKINS_ARTIFACTORY_USERNAME}:${env.JENKINS_ARTIFACTORY_PASSWORD} \
+            -w "%{http_code}" -s -I -o /dev/null \
+            -XGET "https://artifactory.wikia-inc.com/artifactory/api/storage/dockerv2-local/sus/${mediawikiImageName}/${imageTag}"
+          """, returnStdout: true).trim()
+
+          if (status == "200") {
+            imageExists = true
+          }
+
+          if (!imageExists) {
+            // SUS-5284 - make the image a bit smaller
+            sh("cp docker/.dockerignore ..")
+            sh("docker build .. -f $phpDockerFile -t $mediawikiImage:$imageTag")
+            sh("docker push $mediawikiImage:$imageTag")
+          } else {
+            println("Php image tag ($imageTag) already exists")
+          }
+        }
+      }
+    },
+
+    "nginx": {
+      stage("Build and push nginx image") {
+        dir("app") {
+          def imageExists = false
+          def status = sh(script: """
+            curl -u ${env.JENKINS_ARTIFACTORY_USERNAME}:${env.JENKINS_ARTIFACTORY_PASSWORD} \
+            -w "%{http_code}" -s -I -o /dev/null \
+            -XGET "https://artifactory.wikia-inc.com/artifactory/api/storage/dockerv2-local/sus/${mediawikiImageName}-nginx/${imageTag}"
+          """, returnStdout: true).trim()
+
+          if (status == "200") {
+            imageExists = true
+          }
+
+          if (!imageExists) {
+            // SUS-5284 - make the image a bit smaller
+            sh("cp docker/.dockerignore ..")
+            sh("docker build . -f docker/servicesmw/Dockerfile-nginx -t $nginxImage:$imageTag")
+            sh("docker push $nginxImage:$imageTag")
+          } else {
+            println("Nginx image tag ($imageTag) already exists")
+          }
+        }
+      }
+    }
+  )
+
+  stage('Notify Slack Channel publish') {
+    sh "curl -X POST -H 'Content-type: application/json' --data '{ \"text\": \"MediaWiki with app commit <https://github.com/Wikia/app/commit/'${appHash}'|'${appHash}'> and config commit <https://github.com/Wikia/config/commit/'${configHash}'|'${configHash}'> is published by '${buildUser}'\", \"channel\": \"'${env.MW_SANDBOX_K8S_CHANNEL}'\" }' '${env.MW_SANDBOX_K8S_WEBHOOK}'"
```

Comment:
`MW_SANDBOX_K8S_CHANNEL`? which channel is it, I cannot find this env

Reply:
missed that one;)